### PR TITLE
Solr update

### PR DESCRIPTION
**Pull request by fubarhouse**
To accompany https://github.com/govCMS/govCMS8/pull/444

### CODE REVIEW

**Comment by tobybellwood** [comment]
This requires the `"symfony/event-dispatcher": "4.3.11 as 3.4.35",` to be added to composer.json to activate the replacement at root

**Comment by fubarhouse** [comment]
Will add it today.

**Comment by fubarhouse** [comment]
@steveworley,

Noting that I resolved the same issue this morning by changing the values in the `.env.default` file. The Solr base image tag has changed in this release, along withthe search_api_solr version upgrade which needs to be `3.x`.

File: composer.json [hunk 1]
```diff
@@ -29,6 +29,12 @@
         "webflo/drupal-finder": "^1.0.0",
         "webmozart/path-util": "^2.3",
         "drupal/core-recommended": "^8.8",
+        "drupal/fast_404":"1.0.0-alpha4",
```
Comment (pandaskii) [inline composer.json:32]: These changes are against the PR #131  https://github.com/govCMS/govcms8lagoon/pull/131/files

**Pull request merged**